ui/packages/atlasmap-core/src/services/mapping-management.service.ts

```diff
@@ -156,21 +153,30 @@ export class MappingManagementService {
   }
 
   /**
-   * Remove all mappings from the current session (UI and backend service).
+   * Remove all mappings from the current session.
    */
   removeAllMappings(): Promise<boolean> {
-    return new Promise<boolean>((resolve) => {
+    return new Promise<boolean>(async (resolve) => {
       const url: string =
         this.cfg.initCfg.baseAtlasServiceUrl +
         'project/' +
-        this.cfg.mappingDefinitionId +
-        '/mapping';
+        this.cfg.mappingDefinitionId;
```

Comment:
Missing `/mapping`? the REST `DELETE /project/{mappingDefinitionId}` would go this one - https://github.com/atlasmap/atlasmap/blob/main/lib/service/src/main/java/io/atlasmap/service/AtlasService.java#L392
which removes a mapping project, including Documents. In order to remove only mapping entries but keep the Documents, it needs to go to `DELETE porject/{mappingDefinitionId}/mapping` which is here - https://github.com/atlasmap/atlasmap/blob/main/lib/service/src/main/java/io/atlasmap/service/MappingService.java#L106

Reply:
Ah - the `DELETE project/{mappingDefinitionId}/mapping` works - its the `fetchMappings()` in the UI that was getting the mappings based on the original JSON mappings file.  So - I assume I need to do a
`GET project/{mappingDefinitionId}/mapping` ?

Reply:
Yes that's the one to retrieve the mapping - https://github.com/atlasmap/atlasmap/blob/main/lib/service/src/main/java/io/atlasmap/service/MappingService.java#L132

Reply:
Quick tweek to backend service to `setMappingDefinition()` did the trick.